features/profiles.feature

```diff
@@ -22,7 +22,7 @@ Feature: default command line arguments
       module.exports = {
         'default': '-f summary',
         progress: '-f progress',
-        'coffee-snippets': '--coffee'
+        'no-snippets': '-i'
```

Comment:
I know it's not the case in the rest of the file but @aslakhellesoy once recommended me to use full-word flags in code, so that it's easier to understand them just by reading it (and not having to run `--help` on the CLI or read the docs) :)